packages/ui/src/lib/radioButton/RadioButtonGroup.svelte

```diff
@@ -48,16 +57,24 @@
 	{#if !buttonsHidden}
 		<Button variant="text" class="!px-0" on:click={() => (selectedId = '')}>Clear</Button>
 	{/if}
-	{#each options as option}
-		<RadioButton
-			id={option.id}
-			label={option.label}
-			color={option.color}
-			disabled={option.disabled}
-			hint={option.hint}
-			hintLabel={option.hintLabel}
-			bind:selectedId
-			{name}
-		/>
-	{/each}
+	<div
+		class={orientation === 'vertical'
+			? 'flex flex-col space-y-0.25'
+			: orientation === 'horizontal'
+				? 'flex space-x-3'
+				: ''}
+	>
```

Comment:
Does this need to handle the case where orientation is neither `'horizontal'` or '`vertical'`, and is applying no classes a useful thing to do in this case?

 I think we could simplify to:

```suggestion
		class={orientation === 'vertical'
			? 'flex flex-col space-y-0.25'
			: 'flex space-x-3' }
	>
```

Reply:
That makes a lot of sense - thanks, James!